ttnn/cpp/ttnn/operations/conv/conv2d/device/kernels/activation_reader_width_sharded.cpp

```diff
@@ -3,6 +3,7 @@
 // SPDX-License-Identifier: Apache-2.0
 
 #include <stdint.h>
+#include <sys/param.h>
```

Comment:
do we need this include?

Reply:
Co-pilot automatically adds header files. I didn't realize.  I will remove it.